docs/tutorials/adv_tutorial.ipynb

```diff
@@ -215,6 +215,40 @@
     "**Note:** You might notice that the chart line only starts after a couple of steps; this is due to a bug in Charts.js which will hopefully be fixed soon."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Adding PyPlots to visualization\n",
```

Comment:
To be consistent with previous headers I think this can just be **Adding a PyPlot**